Setup Files/install.sh

```diff
@@ -36,7 +41,8 @@ echo " "
 echo "Installing Dependencies"
 echo "======================="
 sudo apt-get dist-upgrade
-sudo apt-get install python-pip git libi2c-dev python-serial python-rpi.gpio i2c-tools python-smbus
+sudo apt-get install -y python-pip git libi2c-dev python-serial python-rpi.gpio i2c-tools python-smbus python-setuptools python-dev build-essential
```

Comment:
@karan259  you'll have to ask Shoban. I simply implemented his steps found in the readme for BrickPy_Python:
*Run sudo apt-get install python-setuptools python-dev build-essential*